rust/feature-flags/src/test_utils.rs

```diff
@@ -137,6 +139,30 @@ pub async fn setup_pg_client(config: Option<&Config>) -> Arc<PgClient> {
     )
 }
 
+pub struct MockPgClient;
```

Comment:
NB: could rename this to "MockErrorPgClient", since it's always in an error state.